packages/scrum-server/src/api/components/planningSession/service.js

```diff
@@ -22,14 +20,13 @@ const save = async req => {
   }
 };
 
-const find = async req => {
-  const planningRoomId = req.params.id;
-  if (!planningRoomId || !ObjectId.isValid(planningRoomId)) {
-    throw Error(i18n.__('apiPlanningSessionInvalidId'));
+const find = async objectId => {
+  if (!objectId || !ObjectId.isValid(objectId)) {
+    throw Error(i18n.__('serviceInvalidId'));
   }
-  const planningSession = await PlanningSession.findOne({ _id: planningRoomId });
+  const planningSession = await PlanningSession.findOne({ _id: objectId });
```

Comment:
why objetId is better than planningRoomId?

Reply:
My bad, i was removing the request in the service and changed the name unintentionally, sorted in the next push